jaeger-context/src/main/java/com/uber/jaeger/context/ThreadLocalTraceContext.java

```diff
@@ -59,4 +53,12 @@ public Span pop() throws EmptyStackException {
   public Span getCurrentSpan() throws EmptyStackException {
     return threadLocal.get().peek();
   }
+  
+  private static class SpanStackThreadLocal extends ThreadLocal<Stack<Span>> {
```

Comment:
I apologize for the confusion, but I was recommending that we maintain the old code and simply add `static` to the nested class. 
I agree that the class names are longer when we do that, but I don't see that being an issue. I don't see the advantage of expanding the visibility scope of this class.